packages/builder/test/fixture-projects/project-with-scripts/failing-status-script.js

```diff
@@ -0,0 +1,4 @@
+
+export default function (runtimeEnv, deployer, accounts) {
+  return 123;
+}
```

Comment:
linter doesn't complain about unused arguments? (it should). 
If it will, just add `_` prefix.

Reply:
It didn't say anything. Also this is in tests, so they may not be checked.